llvm/include/llvm/IR/InstrTypes.h

```diff
@@ -1015,7 +1015,7 @@ struct OperandBundleUse {
   /// has the attribute A.
   bool operandHasAttr(unsigned Idx, Attribute::AttrKind A) const {
     if (isDeoptOperandBundle())
-      if (A == Attribute::ReadOnly || A == Attribute::NoCapture)
```

Comment:
This case is now handled by getCaptureInfo, correct?

Reply:
Yes.